### DISCUSSION TITLE
clear color with partial TextureView

### DISCUSSION THREAD

**Discussion by ghost** [gfx-rs/wgpu#2591] [General]
In Vulkan and DX and OpenGL, I can clear render pass with partial Texture, but in wgpu I can't find way that partial clear ?

In Vulkan for example:

VkRenderPassBeginInfo renderPassBeginInfo = vkTools::initializers::renderPassBeginInfo();
renderPassBeginInfo.renderPass = renderPass;

// this can set part of render, not the full pass

renderPassBeginInfo.renderArea.offset.x = 0;  
renderPassBeginInfo.renderArea.offset.y = 0;
renderPassBeginInfo.renderArea.extent.width = width;
renderPassBeginInfo.renderArea.extent.height = height;

renderPassBeginInfo.clearValueCount = 2;
renderPassBeginInfo.pClearValues = clearValues;

**Comment by kvark** [comment]
More generally, it would be nice to expose render passes into specific area (as opposed to the whole render target space). Also would allows us to do https://github.com/gpuweb/gpuweb/issues/503

**Reply by ghost** [reply]
thanks vey much for give me an answer.
